changelog/unreleased/issue-4768

```diff
@@ -0,0 +1,9 @@
+Enhancement: Allow custom User-Agent to be specified for outgoing requests
+
+Restic uses the default `User-Agent` of the `net/http` package for all
```

Comment:
That is not correct. Most backend libraries inject their own user agent. That wording is also too technical. Just state that restic uses the default user-agent selected by the library used to access the backend.